assets/vue/views/documents/DocumentsList.vue

```diff
@@ -584,4 +598,18 @@ function showUsageDialog() {
   }
   isFileUsageDialogVisible.value = true
 }
+
+function showRecordAudioDialog() {
+  isRecordAudioDialogVisible.value = true
+}
+
+function recordedAudioSaved() {
+  // TODO show a notification about upload success
+  isRecordAudioDialogVisible.value = false
+  onUpdateOptions(options.value)
+}
+
+function recordedAudioNotSaved() {
+  // TODO show a notification about upload failure
+}
```

Comment:
To show the notifications we need to await to the pull request https://github.com/chamilo/chamilo-lms/pull/4729to be merged since the notification composable is made in it